test/development/acceptance-app/component-stack.test.ts

```diff
@@ -50,13 +50,72 @@ describe('Component Stack in error overlay', () => {
 
     await session.waitForAndOpenRuntimeError()
 
-    const expected = `p
-div
-Component
-main`
-    expect((await session.getRedboxComponentStack()).trim()).toStartWith(
-      expected
-    )
+    if (process.env.TURBOPACK) {
+      expect(await session.getRedboxComponentStack()).toMatchInlineSnapshot(`
+        "p
+        div
+        Component
+        main
+        InnerLayoutRouter
+        http (NaN:NaN)
```

Comment:
@kwonoj these `http (NaN:NaN)` parts are unexpected

Reply:
Fixed 💯 The parsing of lines assumed `:` would exist at the end only and `http://myurl/_next/etc:5:10` has `:` for the protocol 👍